src/ExtranetBundle/objects/db/TDataCountryList.class.php

```diff
@@ -13,6 +13,9 @@ class TDataCountryList extends TDataCountryListAutoParent
 {
     /**
      * {@inheritdoc}
+     *
+     * @psalm-suppress AssignmentToVoid, InvalidReturnStatement
+     * @FIXME `Load` is a void method. Saving the parent return value and returning in this method makes no sense.
      */
     public function Load($sQuery, array $queryParameters = array(), array $queryParameterTypes = array())
```

Comment:
`Load` has a `void` return, yet it saves the return value of the parent and returns it itself. This does not seem correct, but I did not want to change it since it's not actually `void` in some cases (it is though, I checked)